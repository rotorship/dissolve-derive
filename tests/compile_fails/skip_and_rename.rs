use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct SkipAndRename {
	#[dissolved(skip, rename = "new_name")]
	field: String,
}

fn main() {}
