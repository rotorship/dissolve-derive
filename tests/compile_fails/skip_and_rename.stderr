error: cannot use rename on skipped field
 --> tests/compile_fails/skip_and_rename.rs:5:2
  |
5 |     #[dissolved(skip, rename = "new_name")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
